index.html

```diff
@@ -264,6 +265,39 @@ <h2>Skills</h2>
             </article>
         </section>
 
+        <section class="contact-section">
+            <article class="contact-article">
+                <p>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+                <form action="https://formspree.io/f/xyyaopza" method="POST" class="contact-form">
+                    <input name="FullName" type="text" placeholder="Full name" required maxlength="30" />
+                    <input name="Email" type="email" placeholder="Email address" required />
+                    <textarea name="Message" rows="4" cols="50" placeholder="Write me something..." maxlength="500" required></textarea>
+                    <div class="contact-button">
+                        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Please implement the submit button color change on mouse hover. 

![image](https://user-images.githubusercontent.com/84156714/236213802-c85c7b06-3508-4fe4-8409-519324b245e8.png)